foreman_rh_cloud.gemspec

```diff
@@ -18,12 +18,12 @@ Gem::Specification.new do |s|
   s.test_files = Dir['test/**/*']
   s.test_files -= Dir['test/**/*.orig']
 
+  s.required_ruby_version = '>= 2.7', '< 4'
```

Comment:
@ekohl interestingly it does not fail

Reply:
That is quite weird. I saw it fail before. I swear I'm not crazy ;)

Reply:
So did I 🤷‍♀️

Reply:
Perhaps the use of theforeman-rubocop actually downgraded it to a version that doesn't have the cop